ansible/roles/ocp-workload-quarkus-cafe-demo-role/tasks/amq-streams-install.yml

```diff
@@ -0,0 +1,55 @@
+- name: Copying over AMQ Subscription
+  template:
+    src: templates/amq-sub.yaml.j2
+    dest:  "{{ config_location }}/amq-sub.yaml"
+    owner: "{{ default_owner }}"
+    group: "{{ default_group }}"
+    mode: 0666
+
+- name: Create Subscription to AMQ
+  command: "oc create -f {{ config_location }}/amq-sub.yaml"
```

Comment:
Please avoid using command and shell.

You should be able to do this using the k8s module.